lib/zipkin-tracer/config.rb

```diff
@@ -38,7 +53,8 @@ def adapter
 
     DEFAULTS = {
       sample_rate: 0.1,
-      service_port: 80
+      service_port: 80,
+      sampled_as_boolean: true
```

Comment:
on the other hand it would change the current behavior without warning.
Maybe leave this default but log a deprecation warning, then after a while we switch the default.